lib/ngp_van/client/supporter_groups.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module NgpVan
+  class Client
+    module SupporterGroups
+      def create_supporter_group(body: {})
+        post(path: 'supporterGroups', body: body)
+      end
+
+      def supporter_group(id:, params: {})
+        get(path: "supporterGroups/#{id}", params: params)
+      end
+
+      def supporter_groups(params: {})
+        get(path: 'supporterGroups', params: params)
+      end
+
+      def add_person_to_support_group(supporter_group_id:, id:)
```

Comment:
should the name of this method and the next one say `supporter_group` rather than `support_group` for consistency?